packages/lb-annotation/src/core/pointCloud/matrix.ts

```diff
@@ -124,6 +130,14 @@ export function pointCloudLidar2image(
     T: [TMatrix14Tuple, TMatrix14Tuple, TMatrix14Tuple];
   },
 ) {
+  if (boxParams.length) {
+    return boxParams.linePointList
+      ?.map((line) => ({
+        type: 'line',
+        pointList: [line],
+      }))
+      .slice(0, boxParams.linePointList.length - 1);
+  }
```

Comment:
The lineTool maybe not need to update `pointCloudLidar2image` ? Why do you need to change this location?